MachineImages/Templates/Debian_i386/Makefile

```diff
@@ -0,0 +1,14 @@
+SHELL = bash
+
+build:
+	packer build Template.json
+	mv -f -v output-debian-i386-vbox/ ../../Artifacts/Debian_i386/
+
+debug-build:
+	PACKER_LOG=1 PACKER_LOG_PATH=Packer.log packer build -debug Template.json
+
+clean:
+	rm -r packer_cache
```

Comment:
adding `-f` to both `rm` commands will prevent failure if files/directories do not exist